data/mods/Magiclysm/Spells/technomancer.json

```
@@ -14,7 +14,7 @@
     "base_energy_cost": 100,
     "energy_source": "MANA",
     "spell_class": "TECHNOMANCER",
-    "difficulty": 1,
+    "difficulty": 2,
```

```suggestion
    "difficulty": 5,
```
...and a bit harder to cast